packages/core/utils/src/dml/__tests__/enum-schema.spec.ts

```diff
@@ -46,4 +46,52 @@ describe("Enum property", () => {
       relationships: [],
     })
   })
+
+  test("use typescript enum as enum property options", () => {
+    enum Roles {
+      ADMIN,
+      WRITER,
+      EDITOR,
+    }
+
+    const property = new EnumProperty(Roles)
+    expectTypeOf(property["$dataType"]).toEqualTypeOf<Roles>()
+
+    expect(property.parse("role")).toEqual({
+      fieldName: "role",
+      dataType: {
+        name: "enum",
+        options: {
+          choices: ["ADMIN", "WRITER", "EDITOR", 0, 1, 2],
```

Comment:
question: Do we handle both types of `choices` formats? This one looks a bit strange